access/resource_table_acl.go

```diff
@@ -0,0 +1,303 @@
+package access
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/compute"
+	"github.com/databrickslabs/databricks-terraform/internal"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// https://docs.databricks.com/security/access-control/table-acls/object-privileges.html#operations-and-privileges
+
+// TableACL defines table access control
+type TableACL struct {
```

Comment:
There are also three other SQL objects that can have acls.
`CATALOG`
`ANY_FILE`
`ANONYMOUS_FUNCTION`
https://docs.databricks.com/security/access-control/table-acls/object-privileges.html

Reply:
are those often used?.. i wonder how to create a resource id for those.